code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -294,3 +294,157 @@
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank)
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
+	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
+	cold_protection = UPPER_TORSO | LOWER_TORSO | LEGS | FEET | ARMS | HANDS
+	siemens_coefficient = 0.9
+
+//These inherit freddy stats
+/obj/item/clothing/head/helmet/fluff/freddy/bonnie
+	desc = "Children's entertainer."
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "bonniehead"
+	item_state = "bonniehead_mob"
+
+/obj/item/clothing/suit/fluff/freddy/bonnie
+	desc = "Children's entertainer."
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "bonniesuit"
+	item_state = "bonniesuit_mob"
+
+/obj/item/clothing/head/helmet/fluff/freddy/foxy
+	desc = "I guess he doesn't like being watched."
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "foxyhead"
+	item_state = "foxyhead_mob"
+
+/obj/item/clothing/suit/fluff/freddy/foxy
+	desc = "I guess he doesn't like being watched."
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "foxysuit"
+	item_state = "foxysuit_mob"
+
+/obj/item/clothing/head/helmet/fluff/freddy/chica
+	desc = "<b><font color=red>LET'S EAT!</font></b>"
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "chicahead"
+	item_state = "chicahead_mob"
+
+/obj/item/clothing/suit/fluff/freddy/chica
+	desc = "<b><font color=red>LET'S EAT!</font></b>"
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "chicasuit"
+	item_state = "chicasuit_mob"
+
+//End event costumes
+
+//scree:Scree
+/obj/item/clothing/head/helmet/space/void/engineering/fluff/screehelm
+	name = "Modified Tajara Helmet"
+	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Has radiation shielding. This one doesn't look like it was made for humans. Its been modified to include headlights."
+
+	icon = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "rig0-scree"
+
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	item_state = "rig0-scree_mob"
+
+//scree:Scree
+/obj/item/clothing/suit/space/void/engineering/fluff/screespess
+	name = "Modified Winged Suit"
+	desc = "A special suit that protects against hazardous, low pressure environments. Has radiation shielding. This one doesn't look like it was made for humans. This one was made with a special personal shielding for someone's wings."
+
+	icon = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "wingedsuit"
+
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	item_state = "wingedsuit"
+
+//scree:Scree
+/obj/item/clothing/under/fluff/screesuit
+	name = "Scree's feathers"
+	desc = "A mop of fluffy blue feathers, the honkmother only knows what kind of bird they originally came from."
+
+	icon = 'icons/vore/custom_clothes_vr.dmi'
+	icon_state = "screesuit"
+
+	icon_override = 'icons/vore/custom_clothes_vr.dmi'
+	item_state = "screesuit_s"
```

Comment:
The _s is added automatically on anything in that slot ('under' aka uniform slot), so if you do this it will look for iconstate screesuit_s_s